### PR TITLE
BUG: Set ElementType for wrapping PyVectorContainer

### PR DESCRIPTION
**Pull request by thewtex**
This is required in the CMake configure_file template to produce the
correct return type.

### CODE REVIEW

**Comment by thewtex** [comment]
This will address the failing test:
https://open.cdash.org/test/373526037

File: Modules/Bridge/NumPy/wrapping/itkPyVectorContainer.wrap [hunk 1]
```diff
@@ -57,6 +59,7 @@ itk_wrap_class("itk::PyVectorContainer")
   endforeach()
   foreach(d ${ITK_WRAP_IMAGE_DIMS})
     itk_wrap_template("${ITKM_UC}${ITKM_O${d}}"  "${ITKT_UC},${ITKT_O${d}}")
+      set(ElementType SL)
       set(PyVectorContainerTypes ${ITKM_UC}${ITKM_O${d}})
```
Comment (tbirdso) [inline Modules/Bridge/NumPy/wrapping/itkPyVectorContainer.wrap:63]: @thewtex In other cases `ElementType` is the same as the second template argument, but here `signed long` and `itk::Offset<dim>` are used. Is this related to data representation in `numpy`?

File: Modules/Bridge/NumPy/wrapping/itkPyVectorContainer.wrap [hunk 2]
```diff
@@ -79,6 +82,7 @@ itk_wrap_class("itk::PyVectorContainer")
   endforeach()
   # used in FastMarchingExtensionImageFilter
   itk_wrap_template("${ITKM_UI}${ITKM_VUC1}"    "${ITKT_UI},${ITKT_VUC1}")
+    set(ElementType UC)
     set(PyVectorContainerTypes ${ITKM_UI}${ITKM_VUC1})
```
Comment (tbirdso) [inline Modules/Bridge/NumPy/wrapping/itkPyVectorContainer.wrap:86]: @thewtex Same question as above -- why is `ElementType` set to `unsigned char` while `PyVectorContainer` is wrapped for elements of type `itk::Vector< unsigned char, 1>`?

File: Modules/Bridge/NumPy/wrapping/itkPyVectorContainer.wrap (hunk 1 above)
Comment (thewtex) [reply Modules/Bridge/NumPy/wrapping/itkPyVectorContainer.wrap:63]: Yes, for `OffSet` the element type is `signed long` per:

https://github.com/InsightSoftwareConsortium/ITK/blob/4f64c6c962272571e5979a284cfcc40e09a2d53e/Modules/Core/Common/include/itkIntTypes.h#L94

But, I will change this because, in practice, it changes based on platform, e.g. it could be "signed long long",

https://github.com/InsightSoftwareConsortium/ITK/blob/4f64c6c962272571e5979a284cfcc40e09a2d53e/Modules/Core/Common/include/itkIntTypes.h#L77

So `ITKT_OT` is the right value:
https://github.com/InsightSoftwareConsortium/ITK/blob/4f64c6c962272571e5979a284cfcc40e09a2d53e/Wrapping/WrapBasicTypes.cmake#L189-L204

File: Modules/Bridge/NumPy/wrapping/itkPyVectorContainer.wrap (hunk 2 above)
Comment (thewtex) [reply Modules/Bridge/NumPy/wrapping/itkPyVectorContainer.wrap:86]: `unsigned char`, i.e. `UC` is the element type of `itk::Vector< unsigned char, 1 >`.

**Review by tbirdso — APPROVED** [review]

File: Modules/Bridge/NumPy/wrapping/PyVectorContainerBuffer.i.in [hunk 3]
```diff
@@ -20,8 +20,8 @@
         else:
             shape   = (itksize,)
 
-        pixelType     = "@PixelType@"
-        numpydatatype = _get_numpy_pixelid(pixelType)
+        element_type     = "@ElementType@"
```
Comment (Leengit) [inline Modules/Bridge/NumPy/wrapping/PyVectorContainerBuffer.i.in:23]: `element_type` is already defined a few lines earlier.  Maybe add a comment somewhere near to indicate why the same variable name is taking on different values in different contexts.  Alternatively, rename one of the variables?

**Review by Leengit — APPROVED** [review]
One question but otherwise it looks good to me.

File: Modules/Bridge/NumPy/wrapping/PyVectorContainerBuffer.i.in (hunk 3 above)
Comment (thewtex) [reply Modules/Bridge/NumPy/wrapping/PyVectorContainerBuffer.i.in:23]: Good point -- changing the above name to `container_element_type` for clarity.

File: Modules/Bridge/NumPy/wrapping/test/itkPyVectorContainerTest.py [hunk 4]
```diff
@@ -89,6 +90,7 @@ def test_NumPyBridge_VectorContainer(self):
         point[2] = 9.0
         v_point.SetElement(1, point)
         arr = itk.array_view_from_vector_container(v_point)
+        print(arr.shape)
```
Comment (dzenanz) [inline Modules/Bridge/NumPy/wrapping/test/itkPyVectorContainerTest.py:93]: Debugging leftover or intentional?

**Review by dzenanz — APPROVED** [review]
LGTM

File: Modules/Bridge/NumPy/wrapping/test/itkPyVectorContainerTest.py (hunk 4 above)
Comment (thewtex) [reply Modules/Bridge/NumPy/wrapping/test/itkPyVectorContainerTest.py:93]: Unintentional :-)
